package.json

```diff
@@ -40,6 +40,8 @@
     "react-dom": ">0.14.0"
   },
   "dependencies": {
+    "react": "^15.4.1",
```

Comment:
Oh, didn't see this before my first comment.  Cant define a react or react-dom version as a dependency because it can result in two versions of react being loaded.  If you need it installed add it as a dev dependency and a peer dependency like we had it.